content/docs/reference/tutorials/kubernetes/tutorial-p8s-rollout.md

```diff
@@ -73,54 +73,52 @@ can see `canary-example-app` and `canary-staging-app` created.
-    +   │  ├─ kubernetes:core:ServiceAccount                           p8s-prometheus-server              created
-    +   │  ├─ kubernetes:core:ServiceAccount                           p8s-prometheus-node-exporter       created
-    +   │  ├─ kubernetes:core:ConfigMap                                p8s-prometheus-alertmanager        created
-    +   │  ├─ kubernetes:rbac.authorization.k8s.io:ClusterRoleBinding  p8s-prometheus-server              created
-    +   │  ├─ kubernetes:core:PersistentVolumeClaim                    p8s-prometheus-server              created
-    +   │  ├─ kubernetes:rbac.authorization.k8s.io:ClusterRoleBinding  p8s-prometheus-kube-state-metrics  created
-    +   │  ├─ kubernetes:core:ConfigMap                                p8s-prometheus-server              created
-    +   │  ├─ kubernetes:core:PersistentVolumeClaim                    p8s-prometheus-alertmanager        created
-    +   │  ├─ kubernetes:core:Service                                  p8s-prometheus-alertmanager        created
-    +   │  ├─ kubernetes:core:Service                                  p8s-prometheus-kube-state-metrics  created
-    +   │  ├─ kubernetes:core:Service                                  p8s-prometheus-pushgateway         created
-    +   │  ├─ kubernetes:core:Service                                  p8s-prometheus-node-exporter       created
-    +   │  ├─ kubernetes:extensions:Deployment                         p8s-prometheus-kube-state-metrics  created
-    +   │  ├─ kubernetes:core:Service                                  p8s-prometheus-server              created
-    +   │  ├─ kubernetes:extensions:Deployment                         p8s-prometheus-pushgateway         created
-    +   │  ├─ kubernetes:rbac.authorization.k8s.io:ClusterRole         p8s-prometheus-kube-state-metrics  created
-    +   │  ├─ kubernetes:extensions:DaemonSet                          p8s-prometheus-node-exporter       created
-    +   │  ├─ kubernetes:extensions:Deployment                         p8s-prometheus-alertmanager        created
-    +   │  ├─ kubernetes:extensions:Deployment                         p8s-prometheus-server              created
-    +   │  └─ kubernetes:rbac.authorization.k8s.io:ClusterRole         p8s-prometheus-server              created
-    +   ├─ kubernetes:apps:Deployment                                  canary-example-app                 created
-    +   └─ kubernetes:apps:Deployment                                  staging-example-app                created
-
-   Diagnostics:
-     pulumi:pulumi:Stack: prometheus-test-p8s
-       info: Checking HTTP metrics
-
-       ---outputs:---
-     + p90ResponseTime: "8221.236"
-
-   info: 2 changes performed:
-       + 2 resources created
-         26 resources unchanged
-   Update duration: 7.362744393s
-
-   Permalink: https://app.pulumi.com/hausdorff/test-p8s/updates/1
-   ```
+        $ pulumi up
```

Comment:
Workaround a bug in Hugo with code fences in lists. (As an aside, I think we can actually work around the bug by using `{{< highlight >}}` in the few places where the bug shows up, allowing us to upgrade to the latest version of Hugo. Will be looking at that separately.)